### PR TITLE
refactor: add utility wrappers for Chain & Web3

### PR DESCRIPTION
**Pull request by geolffreym**
* refactor: add Wrappers for utility types
* test: factory Chain

### CODE REVIEW

File: src/sdk/tests/web3/test_factory.py [hunk 1]
```diff
@@ -1,10 +1,29 @@
...
+    nft_contract,
+    account,
+    chain,
+    _kovan,
+    _rinkeby,
+    Web3Wrapper,
+    ChainWrapper,
+)
+
+from src.sdk.constants import (
```
Comment (aphelionz) [inline src/sdk/tests/web3/test_factory.py:18]: How are these defined? Env vars?

File: src/sdk/tests/web3/test_factory.py [hunk 2]
```diff
@@ -34,7 +53,7 @@ def test_w3_valid_provider():
     """Should return valid w3 wrapper with valid provider"""
 
     valid_chain = "rinkeby"
```
Comment (aphelionz) [inline src/sdk/tests/web3/test_factory.py:55]: I wonder if you should make these a mapping to the numerical chain ID, like rinkeby is 4

File: src/sdk/tests/web3/test_factory.py (hunk 1 above)
Comment (geolffreym) [reply src/sdk/tests/web3/test_factory.py:18]: Exactly, eg.

`KOVAN_PROVIDER = os.getenv("KOVAN_PROVIDER", "")`
all this vars are set from env vars

**Review by aphelionz — CHANGES_REQUESTED** [review]
Python's dynamic typing is kinda biting us here - we should replace the strings like `"rinkeby"` with the numerical chain IDs i.e. `4`

**Pull request closed without merging**